modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/display/context/EditSegmentsEntryDisplayContext.java

```diff
@@ -437,11 +438,20 @@ private Map<String, Object> _getProps() throws Exception {
 			"redirect", getRedirect()
 		).put(
 			"requestMembersCountURL", _getSegmentsEntryClassPKsCountURL()
+		).put(
+			"isSegmentationEnabled", _isSegmentationEnabled()
 		).put(
 			"showInEditMode", _isShowInEditMode()
 		).build();
 	}
 
+	private boolean _isSegmentationEnabled() {
```

Comment:
Hey @YurenaC !!

As we decide in the Pair Programming review, could you change this method, top call directly to 

```
try {
			return _segmentsConfigurationProvider.isSegmentationEnabled(
				companyId);
		}
		catch (ConfigurationException configurationException) {
			_log.error(configurationException);
		}

		return false;
```

We can't make display context depend of other DC, since they should be initialized in the Request, and this could be costly and not needed.
		
Thanks!!

Reply:
Hi @cgoncas its changed but apparently the code wasn't pushed.....
I'll push it asap I finish my current task ;)

Reply:
Now I understand, you fix it in the next commit. 

Could be very useful for the reviewer if the changes are unique pero commit. We can talk a little bit the next week about Commits best practices, are they are described in the PR PLay.

Thanks!!